core/src/main/java/org/apache/accumulo/core/clientImpl/ReplicationOperationsImpl.java

```diff
@@ -63,16 +61,15 @@ public ReplicationOperationsImpl(ClientContext context) {
 
   @Override
   public void addPeer(final String name, final String replicaType)
-      throws AccumuloException, AccumuloSecurityException, PeerExistsException {
```

Comment:
I don't think PeerExistsException gets thrown anywhere.  Sadly its in the public API so it can't be removed right?  Semver in Java has no way to handle exceptions not thrown anymore huh?